rental.rb

```diff
@@ -0,0 +1,19 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
+
+  def add_book(book)
+    @book = book
+    book.rentals << self unless book.rentals.includes?(self)
+  end
+
+  def add_person(person)
+    @person = person
+    person.rentals << self unless person.rentals.include?(self)
+  end
```

Comment:
Well done so far 👍🏼 , but I would sugget assigning book and person in the initializer for code optimisation and to maintain both bidirectional relationships correctly 

` def initialize(date, book, person)
    @book = book
    # *The rental is adding itself to the @book achieving the 'belongs-to' association
    book.rentals << self

    @person = person
    # *The rental is adding itself to the @person the 'belongs-to' association
    person.rentals << self
  end
`